### ISSUE TITLE
Homepage UI redesign

### ISSUE THREAD

**Issue by sophiakc** [openfarmcc/OpenFarm#903]
![new layout it2](https://cloud.githubusercontent.com/assets/3863675/22810617/d7252144-eeee-11e6-8439-65878daf2eb7.jpg)

I'm going to break down the features in different issue...

**Comment by anant-dev** [comment]
@sophiakc  I would like to help make this

**Comment by simonv3** [comment]
That's awesome @anant-dev what particular part of it interests you?

**Comment by anant-dev** [comment]
@simonv3 for starting i could help you with front end and you can guide me for the rest

**Comment by simonv3** [comment]
Cool! do you want to take a stab at the nav bar?

**Comment by nicholasedwards** [comment]
@anant-dev there is at least one CSS rule in app/assets/stylesheets/styles/components/_nav.scss that uses the undesirable `!important` as a work-around to handle the colors of the existing nav bar buttons on the top right of the homepage. If you work on changing the styling to @sophiakc's graphic, it could be a good chance to see if any `!important`s (or entire rules) can be removed to simplify and improve the style of the CSS code.

**Comment by sophiakc** [comment]
Hey @anant-dev how are things going? Do you have any questions, concerns, feedbacks?

**Comment by anyangocynthia** [comment]
@sophiakc Do we need to retain the "Become a member" button for new users? I see its not included in the design

**Comment by sophiakc** [comment]
@anyangocynthia No, the new wireframes is on purpose not including "become a member".

**Comment by anyangocynthia** [comment]
@sophiakc thanks for the feedback. @simonv3 There is a search box on the nav bar , as per the new designs it is not there should I include it?

**Comment by simonv3** [comment]
New designs are canonical!
